target/linux/mediatek/dts/mt7986a-ruijie-rg-x60-pro.dts

```diff
@@ -0,0 +1,432 @@
+// SPDX-License-Identifier: (GPL-2.0 OR MIT)
+
+/dts-v1/;
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/leds/common.h>
+
+#include "mt7986a.dtsi"
+
+/ {
+	compatible = "ruijie,ruijie-rg-x60-pro","mediatek,mt7986a";
+	model = "Ruijie RG-X60 Pro";
+
+	aliases {
+		serial0 = &uart0;
+		led-boot = &sysled;
+		led-failsafe = &atmled;
+		led-running = &sysled;
+		led-upgrade = &atmled;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200n1 loglevel=8 \
+				 tearlycon=uart8250,mmio32,0x11002000";
+	};
+
+	memory {
+		device_type = "memory";
+		reg = <0x00 0x40000000 0x00 0x10000000>;
+	};
+
+	gpio-keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&pio 9 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		mesh {
+			label = "mesh";
+			gpios = <&pio 10 GPIO_ACTIVE_LOW>;
+			linux,code = <BTN_MISC>;
+		};
+	};
+
+	leds-gpio {
+		compatible = "gpio-leds";
+
+		sysled: sysled {
+			label = "white:system";
```

Comment:
New style would be better instead of the label.

```
			color = <LED_COLOR_ID_WHITE>;
			function = LED_FUNCTION_STATUS;
```
https://github.com/torvalds/linux/blob/master/include/dt-bindings/leds/common.h